vizro-core/examples/default/app.py

```diff
@@ -299,18 +299,42 @@ def create_relation_analysis():
             ),
         ],
         controls=[
+            vm.Card(
```

Comment:
Changes will be reverted here - just for demo purposes on how control group headers could be created